users/v1/scripts/main.py

```diff
@@ -0,0 +1,33 @@
+import os
+import csv
+
+import global_variables
+import generate_combos
+import generate_cycles
+import generate_teacher
+
+def valid_combo(combo):
+    if (len(combo) > 0 and combo[0][0] != "#"):
+        return True
+    else:
+        return False
+
+def get_all_combos(descriptions_folder):
+    combos = []
+    for filename in os.listdir(descriptions_folder):
+        with open(os.path.join(descriptions_folder, filename), 'r') as csvfile:
+            csv_content = csv.reader(csvfile, delimiter=',')
+            prefix = ""
+            for row in csv_content:
+                if row and row[0] == "prefix":
+                    prefix = row[1]
+                elif row and valid_combo(row):
+                    combos.append((prefix, row))
+
+    return combos
+
+combos = get_all_combos(global_variables.DESCRIPTIONS_FOLDER)
+
+generate_combos.generate(combos)
```

Comment:
# Not saying this is a good idea, but....
If all your modules will have the same function, you could do something like:
```py
# nitpick: i would remove the "generate_" prefix from the names, it is a bit redundant
MODULES = [generate_combos, generate_cycles, generate_teacher]
for module in MODULES:
    module.generate(combos)
```
At which point you could also use dynamic imports
```py
HERE = Path(__file__).parent    # folder where "main" is
for file in HERE.glob("*.py"):  # for each file in the folder
    if file.stem == "main":
        continue
    module = __import__(file.stem)  # stem = filename without extension
    module.generate(combos)
```